packages/oats-runtime/test/make-from-reflection.spec.ts

```diff
@@ -151,6 +151,58 @@ describe('named', () => {
     const result = fun('aaa').success();
     expect(result).toEqual('aaa');
   });
+
+  it('handles enums with falsy values as discriminators', () => {
+    const atype: Type = {
+      type: 'union',
+      options: [
+        {
+          type: 'object',
+          additionalProperties: false,
+          properties: {
+            name: {
+              value: {
+                type: 'string'
+              },
+              required: true
+            },
+            enabled: {
+              value: {
+                type: 'boolean',
+                enum: [false]
+              },
+              required: true
+            }
+          }
+        },
+        {
+          type: 'object',
+          additionalProperties: false,
+          properties: {
+            other_name: {
+              value: {
+                type: 'string'
+              },
+              required: true
+            },
+            enabled: {
+              value: {
+                type: 'boolean',
+                enum: [true]
+              },
+              required: true
+            }
+          }
+        }
+      ]
+    };
+    const fun = make.fromReflection({ type: 'union', options: [atype] });
+    expect(fun({ tag: true, name: 'aaa' }).success()).toEqual({ tag: false, name: 'aaa' });
```

Comment:
👀 are you sure about the `tag` instead of `enabled` here?

Reply:
copy pasting  is hard